docs/use-github-in-bcgov/github-transition-guide.md

```diff
@@ -84,13 +100,11 @@ The users impacted are:
   - GitHub or OAuth apps to automate the release or testing of your code, including 3rd party security tools 
   - Access tokens and/or SSH keys that your app or team may rely on that will be disconnected when user access is removed, IDE or command line tools and/or automation scripts
 
-### Process  
-
 ### Outside Collaborators
 
 We’re starting with enabling SSO for direct government workers and contractors. After completing the transition for them, we’ll focus on enabling SSO for [outside collaborators](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-outside-collaborators/adding-outside-collaborators-to-repositories-in-your-organization).
 
-Moving forward, **bcgov** GitHub organization needs mandatory and full membership. An option won’t be available to add outside collaborators anymore. 
+Moving forward, the **bcgov** GitHub organization needs mandatory and full membership. The option to add outsider collaborators won’t be available anymore. 
```

Comment:
since the initial draft on this, we've had a few people reach out to ask to be converted to full members. It might be good to provide directions on how to do that here - shiho has them.